src/PointwiseFunctions/AnalyticData/CurvedWaveEquation/PlaneWaveMinkowski.hpp

```diff
@@ -0,0 +1,9 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include "PointwiseFunctions/AnalyticData/CurvedWaveEquation/ScalarWaveGr.tpp"
```

Comment:
Move to cpp.

Reply:
moved. But all other analytic data instantiations in this directory adhere to this include format, i.e. include all `.tpp` and `.hpp` in the Data.hpp file, and only include `Data.hpp` in the `Data.cpp`. Should I change those too? See links [below](https://github.com/sxs-collaboration/spectre/pull/3100#discussion_r657835296)